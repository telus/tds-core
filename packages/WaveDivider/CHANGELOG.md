# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 2.1.3 (2022-02-16)


### Bug Fixes

* gha release job npm publish token ([ce7aacd](https://github.com/telusdigital/tds/commit/ce7aacdd77e54c4f8e48eb31d0c05dc804530324))





## 2.1.2 (2022-02-15)


### Bug Fixes

* gha release job git config ([276ebb6](https://github.com/telusdigital/tds/commit/276ebb6968a0b56c9b87c178f6895a60ae108e71))





## 2.1.1 (2022-02-15)


### Bug Fixes

* gha release job git config ([1ab603d](https://github.com/telusdigital/tds/commit/1ab603d68c36219b0711fc353bc2515b64712ca9))





# [2.1.0](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.13...@tds/core-wave-divider@2.1.0) (2021-09-21)


### Features

* **core-wave-divider:** add react 17 to peer deps ([699ebe2](https://github.com/telusdigital/tds/commit/699ebe20fdfb1912b944ab713c5e72986e6aa403))





## [2.0.13](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.12...@tds/core-wave-divider@2.0.13) (2021-01-15)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.12](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.11...@tds/core-wave-divider@2.0.12) (2020-09-11)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.11](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.10...@tds/core-wave-divider@2.0.11) (2020-09-08)


### Bug Fixes

* **core-wave-divider:** add support for styled-components v5 ([f60bd60](https://github.com/telusdigital/tds/commit/f60bd60b88498312c7b0d31d52edb31161fa9a03))





## [2.0.10](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.9...@tds/core-wave-divider@2.0.10) (2020-09-04)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.9](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.8...@tds/core-wave-divider@2.0.9) (2020-02-03)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.8](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.7...@tds/core-wave-divider@2.0.8) (2020-01-27)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.7](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.6...@tds/core-wave-divider@2.0.7) (2020-01-23)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.6](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.5...@tds/core-wave-divider@2.0.6) (2020-01-22)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.5](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.4...@tds/core-wave-divider@2.0.5) (2019-09-06)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.4](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.3...@tds/core-wave-divider@2.0.4) (2019-05-01)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.3](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.2...@tds/core-wave-divider@2.0.3) (2019-04-22)

**Note:** Version bump only for package @tds/core-wave-divider





## [2.0.2](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.1...@tds/core-wave-divider@2.0.2) (2019-04-03)


### Bug Fixes

* **core-wave-divider:** fix for ie11 compatibility issues ([1913d61](https://github.com/telusdigital/tds/commit/1913d61))





## [2.0.1](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@2.0.0...@tds/core-wave-divider@2.0.1) (2019-03-19)


### Bug Fixes

* **core-wave-divider:** use caret versioning for react peer deps ([d62ce56](https://github.com/telusdigital/tds/commit/d62ce56))





# [2.0.0](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@1.0.1...@tds/core-wave-divider@2.0.0) (2019-03-13)


### Features

* **core-wave-divider:** convert to styled-components ([9f4c552](https://github.com/telusdigital/tds/commit/9f4c552))


### BREAKING CHANGES

* **core-wave-divider:** convert to styled-components





## [1.0.1](https://github.com/telusdigital/tds/compare/@tds/core-wave-divider@1.0.0...@tds/core-wave-divider@1.0.1) (2019-02-06)

**Note:** Version bump only for package @tds/core-wave-divider
