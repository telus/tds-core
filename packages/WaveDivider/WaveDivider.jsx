import React from 'react'

import safeRest from '../../shared/utils/safeRest'

import styles from './WaveDivider.modules.scss'

/**
 * Separate page content blocks.
 *
 * @version ./package.json
 */
const WaveDivider = ({ ...rest }) => (
  <svg
    {...safeRest(rest)}
    className={styles.wave}
    xmlns="http://www.w3.org/2000/svg"
    width="1202"
    height="226"
    viewBox="0 0 1202 226"
  >
    <defs>
      <linearGradient
        id="TDS_WaveDivider_gradient-1"
        x1="600"
        y1="47.26"
        x2="600"
        y2="176.96"
        gradientTransform="matrix(1, 0, 0, -1, 0, 227.8)"
        gradientUnits="userSpaceOnUse"
      >
        <stop offset="0" stopColor="#fff" />
        <stop offset="1" stopColor="#d8d8d8" />
      </linearGradient>
      <linearGradient
        id="TDS_WaveDivider_gradient-2"
        x1="600"
        y1="224.7"
        x2="600"
        y2="70.2"
        gradientTransform="matrix(1, 0, 0, -1, 0, 227.8)"
        gradientUnits="userSpaceOnUse"
      >
        <stop offset="0.19" stopColor="#fff" />
        <stop offset="1" stopColor="#d8d8d8" />
      </linearGradient>
      <linearGradient
        id="TDS_WaveDivider_gradient-3"
        x1="146.95"
        y1="82.92"
        x2="146.95"
        y2="122.08"
        xlinkHref="#TDS_WaveDivider_gradient-1"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-4"
        x1="146.95"
        y1="224.7"
        x2="146.95"
        y2="70.2"
        xlinkHref="#TDS_WaveDivider_gradient-2"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-5"
        x1="1076.11"
        y1="102.31"
        x2="1076.11"
        y2="163.51"
        xlinkHref="#TDS_WaveDivider_gradient-1"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-6"
        x1="1128.83"
        y1="213.27"
        x2="1061.33"
        y2="87.28"
        xlinkHref="#TDS_WaveDivider_gradient-2"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-7"
        x1="28.8"
        y1="115"
        x2="170.73"
        y2="115"
        gradientTransform="matrix(1, 0, 0, -1, 0, 228)"
        gradientUnits="userSpaceOnUse"
      >
        <stop offset="0" stopColor="#fff" />
        <stop offset="1" stopColor="#fff" stopOpacity="0" />
      </linearGradient>
      <linearGradient
        id="TDS_WaveDivider_gradient-8"
        x1="1185.42"
        y1="113.95"
        x2="1041.42"
        y2="115.95"
        xlinkHref="#TDS_WaveDivider_gradient-7"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-9"
        x1="601"
        y1="-2"
        x2="601"
        y2="52.08"
        xlinkHref="#TDS_WaveDivider_gradient-7"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-10"
        x1="600.78"
        y1="222.98"
        x2="600.78"
        y2="160.94"
        gradientTransform="matrix(1, 0, 0, 1, 0, 0)"
        xlinkHref="#TDS_WaveDivider_gradient-7"
      />
      <linearGradient
        id="TDS_WaveDivider_gradient-11"
        x1="601.22"
        y1="64.89"
        x2="601.22"
        y2="2.85"
        gradientTransform="translate(0.01 -0.21) rotate(0.02)"
        xlinkHref="#TDS_WaveDivider_gradient-7"
      />
    </defs>
    <g>
      <g>
        <path
          className={styles.noFill}
          d="M293.89 144.8c24.26 1.32 47.12 2.06 67.8 2.06 193 0 395.64-56 590.52-67C886.68 66.05 821.57 56.1 769.84 56.1 614.41 56.1 452.32 117.3 293.89 144.8Z"
        />
        <g className={styles.opacity3}>
          <path
            className={styles.gradient1}
            d="M1200 146c-57.54-18-153.12-46.11-247.79-66.1-194.87 11.06-397.57 67-590.52 67-20.69 0-43.55-.74-67.8-2.06-46 8-91.66 13.16-136.77 13.16-42.2 0-100.58-6.26-157.12-14.1V224H1200Z"
          />
          <g className={styles.opacity2}>
            <path d="M1200 146c-57.54-18-153.12-46.11-247.79-66.1-194.87 11.06-397.57 67-590.52 67-20.69 0-43.55-.74-67.8-2.06-46 8-91.66 13.16-136.77 13.16-42.2 0-100.58-6.26-157.12-14.1V224H1200Z" />
          </g>
        </g>
        <path
          className={styles.gradient2}
          d="M769.84 56.1c51.74 0 116.84 10 182.38 23.79C973.58 78.68 994.85 78 1016 78c49.13 0 118.77 11.47 184 25.06V0H0V114.49c90.58 12.22 202.25 25.38 293.89 30.31C452.32 117.3 614.41 56.1 769.84 56.1Z"
        />
        <g className={styles.opacity3}>
          <path
            className={styles.gradient3}
            d="M293.89 144.8C202.25 139.8 90.58 126.67 0 114.45V143.9C56.54 151.74 114.92 158 157.12 158 202.23 158 247.91 152.8 293.89 144.8Z"
          />
          <g className={styles.opacity2}>
            <path d="M293.89 144.8C202.25 139.8 90.58 126.67 0 114.45V143.9C56.54 151.74 114.92 158 157.12 158 202.23 158 247.91 152.8 293.89 144.8Z" />
          </g>
        </g>
        <path
          className={styles.gradient4}
          d="M293.89 144.8C202.25 139.8 90.58 126.67 0 114.45V143.9C56.54 151.74 114.92 158 157.12 158 202.23 158 247.91 152.8 293.89 144.8Z"
        />
        <g className={styles.opacity3}>
          <path
            className={styles.gradient5}
            d="M952.21 79.89c94.67 20 190.24 48.11 247.79 66.1V103.05C1134.77 89.47 1065.13 78 1016 78 994.85 78 973.58 78.68 952.21 79.89Z"
          />
          <g className={styles.opacity2}>
            <path d="M952.21 79.89c94.67 20 190.24 48.11 247.79 66.1V103.05C1134.77 89.47 1065.13 78 1016 78 994.85 78 973.58 78.68 952.21 79.89Z" />
          </g>
        </g>
        <path
          className={styles.gradient6}
          d="M952.21 79.89c94.67 20 190.24 48.11 247.79 66.1V103.05C1134.77 89.47 1065.13 78 1016 78 994.85 78 973.58 78.68 952.21 79.89Z"
        />
        <rect className={styles.gradient7} width="180" height="226" />
        <rect className={styles.gradient8} x="1020" width="180" height="226" />
        <rect className={styles.gradient9} y="154" width="1202" height="72" />
      </g>
    </g>
    <g>
      <rect className={styles.gradient10} y="158.3" width="1201.56" height="67.5" />
      <rect
        className={styles.gradient11}
        x="0.44"
        y="0.21"
        width="1201.56"
        height="67.5"
        transform="translate(1202.45 67.7) rotate(179.98)"
      />
    </g>
  </svg>
)

export default WaveDivider
