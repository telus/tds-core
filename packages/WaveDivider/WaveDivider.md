There should be at most 1 `WaveDivider` per page.

Images or artwork should **never** be used in combination with the `WaveDivider`.

```
<WaveDivider />
```
