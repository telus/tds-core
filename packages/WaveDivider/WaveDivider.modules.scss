.wave {
  width: 100%;
}

// SVG
.noFill {
  fill: none;
}

.opacity3 {
  opacity: 0.3;
}

.opacity2 {
  opacity: 0.2;
}

.isolate {
  opacity: 0.5;
  isolation: isolate;
}

.gradient1 {
  fill: url(#TDS_WaveDivider_gradient-1);
}

.gradient2 {
  composes: isolate;

  fill: url(#TDS_WaveDivider_gradient-2);
}

.gradient3 {
  fill: url(#TDS_WaveDivider_gradient-3);
}

.gradient4 {
  composes: isolate;

  fill: url(#TDS_WaveDivider_gradient-4);
}

.gradient5 {
  fill: url(#TDS_WaveDivider_gradient-5);
}

.gradient6 {
  composes: isolate;

  fill: url(#TDS_WaveDivider_gradient-6);
}

.gradient7 {
  fill: url(#TDS_WaveDivider_gradient-7);
}

.gradient8 {
  fill: url(#TDS_WaveDivider_gradient-8);
}

.gradient9 {
  fill: url(#TDS_WaveDivider_gradient-9);
}

.gradient10 {
  fill: url(#TDS_WaveDivider_gradient-10);
}

.gradient11 {
  fill: url(#TDS_WaveDivider_gradient-11);
}
