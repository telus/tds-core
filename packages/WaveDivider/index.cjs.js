const WaveDivider = require('./dist/index.cjs')

module.exports = WaveDivider
