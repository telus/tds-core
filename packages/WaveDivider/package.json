{
  "name": "@tds/core-wave-divider",
  "version": "2.0.4",
  "description": "Separate page content blocks",
  "main": "index.cjs.js",
  "module": "index.es.js",
  "license": "MIT",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/telusdigital/tds.git"
  },
  "publishConfig": {
    "access": "public"
  },
  "author": "TELUS digital",
  "engines": {
    "node": ">=8"
  },
  "bugs": {
    "url": "https://github.com/telusdigital/tds/issues"
  },
  "homepage": "http://tds.telus.com",
  "peerDependencies": {
    "react": "^16.8.0",
    "react-dom": "^16.8.0",
    "styled-components": "^4.1.3"
  },
  "dependencies": {
    "@tds/util-helpers": "^1.1.0",
    "prop-types": "^15.5.10"
  }
}
