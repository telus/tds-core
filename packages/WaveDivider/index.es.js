import WaveDivider from './dist/index.es'

export default WaveDivider
