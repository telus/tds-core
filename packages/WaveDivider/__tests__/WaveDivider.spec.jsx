import React from 'react'
import { shallow, render } from 'enzyme'

import WaveDivider from '../WaveDivider'

describe('WaveDivider', () => {
  const doShallow = (props = {}) => shallow(<WaveDivider {...props} />)

  it('renders', () => {
    const waveDivider = render(<WaveDivider />)

    expect(waveDivider).toMatchSnapshot()
  })

  it('is an svg', () => {
    const waveDivider = doShallow()

    expect(waveDivider).toHaveTagName('svg')
  })

  it('passes additional attributes to the element', () => {
    const waveDivider = doShallow({ id: 'the-id', 'data-some-attr': 'some value' })

    expect(waveDivider).toHaveProp('id', 'the-id')
    expect(waveDivider).toHaveProp('data-some-attr', 'some value')
  })

  it('does not allow custom CSS', () => {
    const waveDivider = doShallow({ className: 'my-custom-class', style: { color: 'hotpink' } })

    expect(waveDivider).not.toHaveProp('className', 'my-custom-class')
    expect(waveDivider).not.toHaveProp('style')
  })
})
