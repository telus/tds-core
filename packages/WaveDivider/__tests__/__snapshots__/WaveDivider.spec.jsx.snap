// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WaveDivider renders 1`] = `
<svg
  class="wave"
  height="226"
  viewBox="0 0 1202 226"
  width="1202"
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <linearGradient
      gradientTransform="matrix(1, 0, 0, -1, 0, 227.8)"
      gradientUnits="userSpaceOnUse"
      id="TDS_WaveDivider_gradient-1"
      x1="600"
      x2="600"
      y1="47.26"
      y2="176.96"
    >
      <stop
        offset="0"
        stop-color="#fff"
      />
      <stop
        offset="1"
        stop-color="#d8d8d8"
      />
    </linearGradient>
    <linearGradient
      gradientTransform="matrix(1, 0, 0, -1, 0, 227.8)"
      gradientUnits="userSpaceOnUse"
      id="TDS_WaveDivider_gradient-2"
      x1="600"
      x2="600"
      y1="224.7"
      y2="70.2"
    >
      <stop
        offset="0.19"
        stop-color="#fff"
      />
      <stop
        offset="1"
        stop-color="#d8d8d8"
      />
    </linearGradient>
    <linearGradient
      href="#TDS_WaveDivider_gradient-1"
      id="TDS_WaveDivider_gradient-3"
      x1="146.95"
      x2="146.95"
      y1="82.92"
      y2="122.08"
    />
    <linearGradient
      href="#TDS_WaveDivider_gradient-2"
      id="TDS_WaveDivider_gradient-4"
      x1="146.95"
      x2="146.95"
      y1="224.7"
      y2="70.2"
    />
    <linearGradient
      href="#TDS_WaveDivider_gradient-1"
      id="TDS_WaveDivider_gradient-5"
      x1="1076.11"
      x2="1076.11"
      y1="102.31"
      y2="163.51"
    />
    <linearGradient
      href="#TDS_WaveDivider_gradient-2"
      id="TDS_WaveDivider_gradient-6"
      x1="1128.83"
      x2="1061.33"
      y1="213.27"
      y2="87.28"
    />
    <linearGradient
      gradientTransform="matrix(1, 0, 0, -1, 0, 228)"
      gradientUnits="userSpaceOnUse"
      id="TDS_WaveDivider_gradient-7"
      x1="28.8"
      x2="170.73"
      y1="115"
      y2="115"
    >
      <stop
        offset="0"
        stop-color="#fff"
      />
      <stop
        offset="1"
        stop-color="#fff"
        stop-opacity="0"
      />
    </linearGradient>
    <linearGradient
      href="#TDS_WaveDivider_gradient-7"
      id="TDS_WaveDivider_gradient-8"
      x1="1185.42"
      x2="1041.42"
      y1="113.95"
      y2="115.95"
    />
    <linearGradient
      href="#TDS_WaveDivider_gradient-7"
      id="TDS_WaveDivider_gradient-9"
      x1="601"
      x2="601"
      y1="-2"
      y2="52.08"
    />
    <linearGradient
      gradientTransform="matrix(1, 0, 0, 1, 0, 0)"
      href="#TDS_WaveDivider_gradient-7"
      id="TDS_WaveDivider_gradient-10"
      x1="600.78"
      x2="600.78"
      y1="222.98"
      y2="160.94"
    />
    <linearGradient
      gradientTransform="translate(0.01 -0.21) rotate(0.02)"
      href="#TDS_WaveDivider_gradient-7"
      id="TDS_WaveDivider_gradient-11"
      x1="601.22"
      x2="601.22"
      y1="64.89"
      y2="2.85"
    />
  </defs>
  <g>
    <g>
      <path
        class="noFill"
        d="M293.89 144.8c24.26 1.32 47.12 2.06 67.8 2.06 193 0 395.64-56 590.52-67C886.68 66.05 821.57 56.1 769.84 56.1 614.41 56.1 452.32 117.3 293.89 144.8Z"
      />
      <g
        class="opacity3"
      >
        <path
          class="gradient1"
          d="M1200 146c-57.54-18-153.12-46.11-247.79-66.1-194.87 11.06-397.57 67-590.52 67-20.69 0-43.55-.74-67.8-2.06-46 8-91.66 13.16-136.77 13.16-42.2 0-100.58-6.26-157.12-14.1V224H1200Z"
        />
        <g
          class="opacity2"
        >
          <path
            d="M1200 146c-57.54-18-153.12-46.11-247.79-66.1-194.87 11.06-397.57 67-590.52 67-20.69 0-43.55-.74-67.8-2.06-46 8-91.66 13.16-136.77 13.16-42.2 0-100.58-6.26-157.12-14.1V224H1200Z"
          />
        </g>
      </g>
      <path
        class="gradient2"
        d="M769.84 56.1c51.74 0 116.84 10 182.38 23.79C973.58 78.68 994.85 78 1016 78c49.13 0 118.77 11.47 184 25.06V0H0V114.49c90.58 12.22 202.25 25.38 293.89 30.31C452.32 117.3 614.41 56.1 769.84 56.1Z"
      />
      <g
        class="opacity3"
      >
        <path
          class="gradient3"
          d="M293.89 144.8C202.25 139.8 90.58 126.67 0 114.45V143.9C56.54 151.74 114.92 158 157.12 158 202.23 158 247.91 152.8 293.89 144.8Z"
        />
        <g
          class="opacity2"
        >
          <path
            d="M293.89 144.8C202.25 139.8 90.58 126.67 0 114.45V143.9C56.54 151.74 114.92 158 157.12 158 202.23 158 247.91 152.8 293.89 144.8Z"
          />
        </g>
      </g>
      <path
        class="gradient4"
        d="M293.89 144.8C202.25 139.8 90.58 126.67 0 114.45V143.9C56.54 151.74 114.92 158 157.12 158 202.23 158 247.91 152.8 293.89 144.8Z"
      />
      <g
        class="opacity3"
      >
        <path
          class="gradient5"
          d="M952.21 79.89c94.67 20 190.24 48.11 247.79 66.1V103.05C1134.77 89.47 1065.13 78 1016 78 994.85 78 973.58 78.68 952.21 79.89Z"
        />
        <g
          class="opacity2"
        >
          <path
            d="M952.21 79.89c94.67 20 190.24 48.11 247.79 66.1V103.05C1134.77 89.47 1065.13 78 1016 78 994.85 78 973.58 78.68 952.21 79.89Z"
          />
        </g>
      </g>
      <path
        class="gradient6"
        d="M952.21 79.89c94.67 20 190.24 48.11 247.79 66.1V103.05C1134.77 89.47 1065.13 78 1016 78 994.85 78 973.58 78.68 952.21 79.89Z"
      />
      <rect
        class="gradient7"
        height="226"
        width="180"
      />
      <rect
        class="gradient8"
        height="226"
        width="180"
        x="1020"
      />
      <rect
        class="gradient9"
        height="72"
        width="1202"
        y="154"
      />
    </g>
  </g>
  <g>
    <rect
      class="gradient10"
      height="67.5"
      width="1201.56"
      y="158.3"
    />
    <rect
      class="gradient11"
      height="67.5"
      transform="translate(1202.45 67.7) rotate(179.98)"
      width="1201.56"
      x="0.44"
      y="0.21"
    />
  </g>
</svg>
`;
